hackathon_site/event/api_views.py

```diff
@@ -163,6 +164,23 @@ def patch(self, request, *args, **kwargs):
         return self.partial_update(request, *args, **kwargs)
 
 
+class CreateProfileView(
+    generics.GenericAPIView, mixins.CreateModelMixin,
+):
+    serializer_class = CreateProfileSerializer
+    queryset = Profile.objects.all()
+
+    def post(self, request, *args, **kwargs):
+        # create a new profile for the user
+        profile = Profile.objects.create(user=self.request.user)
+        team = profile.team
+
```

Comment:
@YuYing-Liang Yuying, do we need to check whether the profile exists or not here before creating the profile object?